docs/en/concepts-and-designs/mal.md

```diff
@@ -0,0 +1,188 @@
+its matching sample in the right-hand sample family. A new sample family with empty name will be generated.
+Only the matched tags will be reserved. Samples for which no matching sample in the right-hand sample family are not in the result.
+
+Another sample family `instance_trace_analysis_error_count` is 
+
+```
+instance_trace_analysis_error_count{region="us-west",az="az-1"} 20
+instance_trace_analysis_error_count{region="asia-north",az="az-1"} 11 
+```
+
+Example expression:
+
+```
+instance_trace_analysis_error_count / instance_trace_count
+```
+
+This returns a result sample family containing the error rate of trace analysis. The samples with region us-west and az az-3 
+have no match and will not show up in the result:
+
+```
+{region="us-west",az="az-1"} 0.8  // 20 / 100
+{region="asia-north",az="az-1"} 0.3333  // 11 / 33
+```
+
+### Aggregation Operation
+
+Sample family supports the following aggregation operations that can be used to aggregate the samples of a single sample family,
+resulting in a new sample family of fewer samples(even single one) with aggregated values:
+
+ - sum (calculate sum over dimensions)
+ - min (select minimum over dimensions)
+ - max (select maximum over dimensions)
+ - avg (calculate the average over dimensions)
+ 
+These operations can be used to aggregate over all label dimensions or preserve distinct dimensions by inputting `by` parameter. 
+
+```
+<aggr-op>(by: <tag1, tag2, ...>)
+```
+
+Example expression:
+
+```
+instance_trace_count.sum(by: ['az'])
+```
+
+will output a result:
+
+```
+instance_trace_count{az="az-1"} 133 // 100 + 33
+instance_trace_count{az="az-3"} 20
+```
+
+### Function
+
+`Duraton` is a textual representation of a time range. The formats accepted are based on the ISO-8601 duration format {@code PnDTnHnMn.nS}
+ with days considered to be exactly 24 hours.
+
+Examples:
+ - "PT20.345S" -- parses as "20.345 seconds"
+ - "PT15M"     -- parses as "15 minutes" (where a minute is 60 seconds)
+ - "PT10H"     -- parses as "10 hours" (where an hour is 3600 seconds)
+ - "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+ - "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+ - "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+ - "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+ - "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+
+#### increase
+`increase(Duration)`. Calculates the increase in the time range.
+
+#### rate
+`rate(Duration)`. Calculates the per-second average rate of increase of the time range.
+
+#### irate
+`irate()`. Calculates the per-second instant rate of increase of the time range.
+
+#### tag
+`tag({allTags -> })`. Update tags of samples. User can add, drop, rename and update tags.
+
+#### histogram
+`histogram(le: '<the tag name of le>')`. Transforms less based histogram buckets to meter system histogram buckets. 
+`le` parameter hints the tag name of a bucket. 
+
+#### histogram_quantile
```

Comment:
Why we are using `quantile`? What parameter format you are going to use? 
According to your `φ-quantile (0 ≤ φ ≤ 100)` expression, you still should use `percentile` here.  Take a look on this https://stats.stackexchange.com/questions/156778/percentile-vs-quantile-vs-quartile

Reply:
Thanks for clarifying it, will fix the term.

Reply:
done